package.json

```diff
@@ -17,6 +17,7 @@
     "test:e2e:iphone8": "detox test -c iphone8.sim --loglevel=warn",
     "i18n:extract": "i18next",
     "test": "jest test",
+    "update-snapshots": "jest --updateSnapshot",
```

Comment:
I'm not sure if this is something we should encourage regularly. I'll defer to @tstirrat on that, but it seems like this encourages us to disregard changes in the snapshots, which negates their purpose.

Reply:
@efalkner This was necessary as we refactored all text and new components were added 
@tstirrat Can confirm

Reply:
I think snapshot testing isn't great, and would like to do better/different testing.

Specifically this script is a helpful script while we're in this state. So I'd say, ok, cool.

We should discuss elsewhere about doing better testing, and when we have a better strategy, we can purge snapshot testing then  🔥